crates/nu-command/tests/commands/do_.rs

```diff
@@ -122,8 +122,9 @@ fn ignore_error_with_too_much_stderr_not_hang_nushell() {
 }
 
 #[test]
+#[ignore]
```

Comment:
You don't need to ignore the test, I think it can be fixed easily

Reply:
Done.